worlds/peaks_of_yore/__init__.py

```diff
@@ -0,0 +1,172 @@
+                                                (not self.options.disable_solemn_tempest) or v != 37}
+
+            self.artefacts_in_pool.extend(get_location_names_by_type(3, "Artefact"))
+            self.peaks_in_pool.append("Great Bulwark")
+            if not self.options.disable_solemn_tempest:
+                self.peaks_in_pool.append("Solemn Tempest")
+
+            expert_region.add_locations(expert_locations, PeaksOfYoreLocation)
+            cabin_region.connect(expert_region, "Expert Book", lambda state: state.has("Expert Book", self.player))
+            self.location_count += len(expert_region.locations)
+
+    def create_items(self) -> None:
+        # order: books, tools, ropes, bird seeds, artefacts, fill rest with extra Items
+        starting_book: Item = self.create_item(
+            ["Fundamentals Book", "Intermediate Book", "Advanced Book", "Expert Book"][
+                self.options.starting_book.value])
+        self.multiworld.push_precollected(starting_book)
+
+        if self.options.start_with_barometer:
+            self.multiworld.push_precollected(self.create_item("Barometer"))
+
+        remaining_items = self.location_count
+
+        for book in [item for item in full_item_list if item.type == "Book"]:
+            if remaining_items > 0:
+                item = self.create_item(book.name)
+                if starting_book.name == item.name:
+                    continue
+                self.multiworld.itempool.append(item)
+                remaining_items -= 1
+
+        for tool in [item for item in full_item_list if item.type == "Tool"]:
+            if remaining_items > 0 and (tool.name != "Barometer" or not self.options.start_with_barometer):
+                self.multiworld.itempool.append(self.create_item(tool.name))
+                remaining_items -= 1
+
+        remaining_items -= 1
+        self.multiworld.itempool.append(self.create_item("Progressive Crampons"))
+
+        for rope in [item for item in full_item_list if item.type == "Rope"]:
+            if remaining_items > 0:
+                self.multiworld.itempool.append(self.create_item(rope.name))
+                remaining_items -= 1
+
+        for birdSeed in [item for item in full_item_list if item.type == "Bird Seed"]:
+            if remaining_items > 0:
+                self.multiworld.itempool.append(self.create_item(birdSeed.name))
+                remaining_items -= 1
+
+        for artefact in [item for item in full_item_list if item.type == "Artefact"]:
+            if remaining_items > 0:
+                self.multiworld.itempool.append(self.create_item(artefact.name))
+                remaining_items -= 1
+
+        self.multiworld.itempool += [self.create_extra_item() for _ in range(remaining_items)]
+
+    def set_rules(self) -> None:
+        if self.options.goal.value == Goal.option_all_artefacts:
+            print(self.artefacts_in_pool)
```

Comment:
Please remove debug prints

Reply:
An alternative is `logging.debug(...)`, at which point you can see the messages by passing --log_level=debug as an arg to Generate.py